ledger/record/index_serialize.go

```diff
@@ -0,0 +1,35 @@
+package record
+
+import (
+	"bytes"
+
+	"golang.org/x/crypto/sha3"
+	"github.com/ugorji/go/codec"
+)
+
+// HashLifelineIndex returns 28 bytes of SHA3 hash
+func HashLifelineIndex(buf []byte) Hash {
+	return sha3.Sum224(buf)
```

Comment:
it's a better use hash.SHA3hash224 implementation, according to the single responsibility principle.

i.e. to implement WriteHash method on LifelineIndex